package.json

```diff
@@ -32,6 +32,7 @@
     "forwarder": "node ./development/static-server.js ./node_modules/@metamask/forwarder/dist/ --port 9010",
     "dapp-forwarder": "concurrently -k -n forwarder,dapp -p '[{time}][{name}]' 'yarn forwarder' 'yarn dapp'",
     "test:unit": "node ./test/run-unit-tests.js --mocha --jestGlobal --jestDev",
+    "test:unit:fuzz": "JAZZER_FUZZ=1 jest --config='./test/fuzz/jest.config.js' --testNamePattern=\"Fuzz\"",
```

Comment:
does the jest config file need to be added `'./test/fuzz/jest.config.js'`?

Reply:
That was just something to get up and running quickly, I believe we can move it into the top level jest config file with the use of projects.